pandas/formats/style.py

```diff
@@ -444,15 +444,17 @@ def apply(self, func, axis=0, subset=None, **kwargs):
 
         Parameters
         ----------
-        func: function
-        axis: int, str or None
+        func : function
+            ``func`` should take a Series or DataFrame (depending
```

Comment:
Ah, I thought we decided to use double ticks since it looks nicer in our HTML :)